lib/libutils/ext/include/mempool.h

```diff
@@ -21,6 +21,15 @@ struct mempool;
 
 #define MEMPOOL_ALIGN	__alignof__(long)
 
+#if defined(__KERNEL__)
+/*
+ * System wide memory pool for temporary memory allocation, all allocations
+ * must be freed before allocating function returns in order to avoid
+ * dead-lock.
+ */
+extern struct mempool *mempool_default;
```

Comment:
"default" does not describe the purpose nor the constraint on the usage of this pool. How about `mempool_transient` or `mempool_temp` maybe?

Reply:
All mempool allocations are temporary, that part of the deal with this interface. Perhaps I should try to clarify that in the description of the interface instead.